x/tokenomics/keeper/update_relay_mining_difficulty_test.go

```diff
@@ -213,7 +216,11 @@ func TestComputeNewDifficultyHash(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
 			result := keeper.ComputeNewDifficultyTargetHash(tt.numRelaysTarget, tt.relaysEma)
-			require.Equal(t, result, tt.expectedDifficultyHash)
+
+			require.Lessf(t, bytes.Compare(result, tt.expectedDifficultyHash), 1,
```

Comment:
Not sure if this works: Does `require.Lessf(t, result, target)` not work?

Reply:
Unfortunately, `require.Lessf()` is not that clever. We have to use `bytes.Compare()`.